src/Dropdown/__tests__/Dropdown-test.js

```diff
@@ -270,4 +266,65 @@ describe("Dropdown", function() {
       expect(instance.getSelectedID()).toEqual("foo");
     });
   });
+
+  describe.only("#getSelectedItem", function() {
+    it.only("returns item for initialID", function() {
+      expect(this.instance.getSelectedItem()).toEqual(MockDropdownList[1]);
+    });
+    it("returns null if nothing is given", function() {
+      var instance = TestUtils.renderIntoDocument(
+        <Dropdown
+          buttonClassName="button dropdown-toggle"
+          dropdownMenuClassName="dropdown-menu"
+          dropdownMenuListClassName="dropdown-menu-list"
+          items={MockDropdownList}
+          transition={false}
+          wrapperClassName="dropdown"
+        />,
+        document.body
+      );
+      expect(instance.getSelectedItem()).toEqual(null);
+    });
+    it("shows placeholder when initialized with given trigger", function() {
```

Comment:
Do these tests belong here? 🤔

Reply:
I could create a `ButtonTrigger-test.js` 🤔

Reply:
Please do this